rust/sealevel/client/src/cmd_utils.rs

```diff
@@ -150,7 +150,9 @@ fn attempt_program_deploy(
     ];
 
     let compute_unit_price_str = compute_unit_price.to_string();
-    command.extend(vec!["--with-compute-unit-price", &compute_unit_price_str]);
+    if compute_unit_price > 0 {
```

Comment:
LGTM, a slightly more reliable fix would be to pass `local_domain` as an arg to `attempt_program_deploy` and not set the flag if the local domain is eclipse

Reply:
ah good idea, i'll just prevent bidding up on eclipse for now